app/src/main/java/com/example/atheneum/activities/AddBookActivity.java

```diff
@@ -44,8 +54,6 @@
 public class AddBookActivity extends AppCompatActivity {
 
     private Context context;
-    private User owner;
-    private Book newBook;
     String title;
```

Comment:
Small change but can we make these private?

Reply:
They aren't being used anymore so I deleted them